### DISCUSSION TITLE
[KtonDAO] [Pre Temp Check] Build New Arm for KtonDAO: Crypto Index Fund Curator

### DISCUSSION THREAD

**Discussion by hackfisher** [ktondao/KtonDAO#32] [Q&A]
Currently, one of KtonDAO's primary roles can be summarized as providing instant lock-up incentives (minting KTON) for RingDAO, in exchange for staking incentives offered by RingDAO. However, this model faces scalability challenges in attracting more clients like RingDAO and poses risks to KtonDAO's monetary policy and inflation, given its reliance on issuing new KTON as part of the incentive structure. Therefore, it is necessary to explore new initiatives for KtonDAO’s growth.

Thanks to the [suggestion](https://github.com/orgs/ringecosystem/discussions/8#discussioncomment-11334181) from @SasoLithops,  

Here is an idea for the new arm of KtonDAO, creating a **Crypto Index Fund** (based on Cow-AMM pools) consisting of tokens from various communities. This fund would cater to users interested in multiple interconnected communities, fostering inter-community relationships and improving liquidity. Inspiration has been drawn from [ndex Fund concept](https://app.devcon.org/schedule/8DCP9K).

## Concept Outline

Imagine a **Crypto Index Fund** curated through community votes to determine which tokens are included or removed. This democratic process could involve all participating communities, with **KTON** as the governance/voting token (addressing its current lack of additional utility) and **RING** used for minting the index token.

### Key Features
- Both **RING** and **KTON** are default components of the index (with adjustable percentages).  
- A portion of the **RING** used for minting tokens could be:
  - Burned, reducing supply.
  - Allocated to the treasury.
  - Used as governance incentives for fund curators.
- Including liquid staked tokens in the index offers additional staking rewards, making this a focal area for development.
- Part of the fees generated from minting the index token would fund further development and maintenance of the ecosystem.

---

## Next Steps

Before proceeding, we need to review **KtonDAO’s current reserves**:  
[KtonDAO Treasury](https://explorer.darwinia.network/address/0x08837De0Ae21C270383D9F2de4DB03c7b1314632)

Below is an initial high-level roadmap for planning and execution:

### 1. **KtonDAO Website**
- **TBD**: Seek support from RingDAO’s education team and DCDAO’s web developers.  
- Community contributions are essential for content creation and review.

### 2. **Tools and Integration**
- For other networks: Consider using **Balancer Protocol** for the Index Fund implementation.  
Refer:
[1] https://balancer.fi/pools/cow
[2] https://balancer.fi/pools/arbitrum/cow/0xaef24dd15ad265319a5f0bfa53bf3cd328375b7b
- For Darwinia: As far as I know, Helixbox is interested in building an AMM on Darwinia. We could explore collaborating with them and inquire whether they are interested in deploying Balancer pools on Darwinia. It would be even better if they could provide a pool management interface.

### 3. **Curator Proposal and Index Fund Setup**
- Tasks include pool creation, initial liquidity provisioning, promotion, attracting additional liquidity, and providing ongoing support for user convenience.
- Given KtonDAO’s current limited reserves, we propose reallocating a higher proportion of KTON staking rewards to support these efforts once this initiative is finalized.

---

## Call to Action

These are preliminary ideas and directions that require detailed planning and project management. Success depends on collaboration between ecosystem participants and external service providers.

We encourage **KTON holders** to actively contribute and provide feedback. Delegates of KtonDAO should refine these ideas and offer input for the next **Temp Check** to align the community on a clear path forward.

---

Note: Please note that this is merely a concept for discussion and does not imply that the above content has been adopted as part of the DAO’s plan.

**Comment by SasoLithops** [comment]
Thank you @hackfisher for putting this together. I have many ideas on this but need a bit more time to put it in reasonable thread. I will do that in next few days. Looking fwd to community input. 42000+ Kton staked, we need to get those KTON stakers expressing their opinion here and join the KTON DAO governance.

**Reply by sonoferin2000** [reply]
same here, looking forward to the weekend, will try to digest this all and come up with my own input.  its good to have discussion.

**Comment by SasoLithops** [comment]
Absolutely agree with everything on **Concept Outline**


As for **Next Steps** I have some proposals to add/change.

**1. KtonDAO Website**

Website can be created by community and also managed by it if we have some website builder among us. I would much rather we pay her/him then random dev. 

**2. Tools and Integration**

I have checked https://balancer.fi/pools/cow it has very few tokens with decent liquidity. Also BTC pools dont have much liquidity. 

I would prefer we take advantage of polaris( check here:  https://x.com/Polaris_App ) which will launch in H1/2025. This also gives us time to develop well, before product launch. We have open TG channel with Osmosis which team is building Polaris app. This Polaris app will not have its own liquidity but will pull it from ecosystems that have enough of it. It will not compete with native chain dexes but use their best dex liquidity. That would give us acccess to many pools with large ammount of liquidity. 

Pools in Darwinia if created, would lack liquidity imho.

I would like to hear how hard it would be to build connection to all that? 

**3. Curator Proposal and Index Fund Setup**

Addressing this will take more time. I will share my experience with one index fund I helped set up before and what I have learned. I will write this over next week... 

**Important**

I also think we should asap address how we would fund KTON DAO treasury with not just RING. So we are ready when bear returns. I will work on that too. It will be much easier once we have a product Cross chain Index Fund to sell as different tokens should arrive in treasury. But more on this next week. I have half written so far...

**Reply by sonoferin2000** [reply]
I support this idea, but I have some concerns regarding the transaction fees associated with bridging. Our network is currently very inactive, and token prices could easily deviate from their actual market value. I believe it would be better to implement this on a network with ample liquidity for the tokens we need, as this would eliminate the need for repeated bridging fees and provide access to a larger user base.

If we proceed on our own network, I anticipate that people might sell their RING rewards to buy into the index, which could drive down the token's price. I think this initiative should be developed by the DAO as a revenue-generating mechanism, allowing us to earn from fees. Including our token in the mix could help anchor its value to more stable tokens.

However, I feel this should be presented to the broader crypto community, possibly on Polkadot, where there’s more capital and activity. This is just my opinion. this is not something we need to rush into. but im definitely on board with the idea.

**Reply by hackfisher** [reply]
Added an RPF from https://github.com/ringecosystem/collaboration/issues/99

**Comment by sonoferin2000** [comment]
How would this operate?would it operate like a banacer liquidity pool? or would it be more like a tokenset? Would it accept multi token deposits,or allow a single token, such as eth or some other, and the contract handle purchasing the tokens needed from that amount? Or will people supply all their own tokens in the proper combination? The easier we make it to interact with, the greater the chance it will gain traction. allow for option to simply provide 1 token and the contract handle all the sourcing would massively simplify things from an end user perspective, and we could possibly charge a fee saving them time by using that. just a thought. If this becomes a success, we might also want to create a Kusama version which could include ckton.

**Reply by hackfisher** [reply]
https://github.com/orgs/ktondao/discussions/38

Yes, I agree we need to **KEEP IT SIMPLE** and can only do/start with some due to risk/solution/feasibility limitations.

I would suggest we develop something we will use by ourselves, market and user driven: If no one uses it, it’s meaningless.

**Comment by SasoLithops** [comment]
If technically feasible I absolutely agree with @sonoferin2000 to make it easier for user to interact. I also agree with @hackfisher to KEEP IT SIMPLE. 

I liked this idea where one could mint our index directly on web page example:

https://index.ion.wtf/mint

We could use multiple assets to be used for buying(can be added in time) and as @sonoferin2000 mentioned some % of fees could go to treasury. This could make our treasury multichain. Exactlly what we aim for.

**Reply by sonoferin2000** [reply]
hopefully it wouldnt be geo-fenced like https://index.ion.wtf/mint .  everyone hates Americans. I promise, we arent all bad.
